docs/sections/calculations/workflows.md

```diff
@@ -2,7 +2,7 @@
 
 # Running workflows
 
-AiiDA can help you run individual calculations, but it is really designed to help you run workflows that involve several calculations, while automatically keeping track of the provenance for full reproducibility.
+AiiDA can help you run individual calculations, but it is really designed to help you run workflows that involve several of them, while automatically keeping track of the provenance for full reproducibility.
```

Comment:
I personally prefer the explicitness of the original sentence here...

Reply:
I went back to being explicit but used "sub-processes" to be less repetitive.